### ISSUE TITLE
Load Balanced Web Service - Healthcheck feature gap

### ISSUE THREAD

**Issue by afgallo** [aws/copilot-cli#2067]
I've reported this issue on the gitter channel and am now raising it here as @efekarakus mentioned it would help you guys keep track of it. Thanks again for your amazing work on the copilot CLI, this is really good stuff!!

___
I'm confused why the `healthcheck` config below for a Load Balanced Web Service isn't working. I don't actually get any errors, however, copilot reports this service health status as `UNKNOWN`:

![image](https://user-images.githubusercontent.com/9999415/111412646-e5e37080-8730-11eb-96cc-5b4b89b3fae1.png)

My manifest configuration:

```
...

type: Load Balanced Web Service

http:
  path: '/'
  healthcheck:
    path: '/hc'
    healthy_threshold: 3
    unhealthy_threshold: 2
    interval: 15s
    timeout: 10s
 
 ...
```

More context on the conversation below:

<img width="595" alt="image" src="https://user-images.githubusercontent.com/9999415/111412815-3d81dc00-8731-11eb-991e-5246bf9571eb.png">

**Comment by coopsmoss** [comment]
I am also experiencing this.

**Comment by efekarakus** [comment]
Hi @afgallo and @coopsmoss ! This is now resolved in v1.8.0 🥳  https://github.com/aws/copilot-cli/releases/tag/v1.8.0
You can use `copilot svc status` to view the target group heath.
